src/components/Input/index.tsx

```diff
@@ -32,6 +32,7 @@ export const Input = React.forwardRef<HTMLInputElement, InputCustomProps>(
       maxCount,
       handleChange,
       dataId = '',
+      isValid,
```

Comment:
isn't it same as hasError?

Reply:
I think no, because in the initial state, hasError is true, so the form is valid.